frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -93,18 +150,121 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
             <p>{uninstallMessage}</p>
           </>
         )}
+        <Operands
+          csvName={csv.metadata.name}
+          operands={operands}
+          operandsLoaded={operandsLoaded}
+          deleteAllOperands={deleteAllOperands}
+          setDeleteAllOperands={setDeleteAllOperands}
+          cancel={cancel} // to cancel modal when clicking on operand links
+        />
       </ModalBody>
       <ModalSubmitFooter
         inProgress={inProgress}
-        errorMessage={errorMessage}
+        errorMessage={errorMessage || loadOperandsErrorMessage}
         cancel={cancel}
         submitDanger
         submitText={t('olm~Uninstall')}
+        submitDisabled={!operandsLoaded}
       />
     </form>
   );
 };
 
+const OperandsTable: React.FC<OperandsTableProps> = (props) => {
+  const { t } = useTranslation();
+  const Header = () => {
+    return [
+      {
+        title: t('public~Name'),
+        sortField: 'metadata.name',
+        transforms: [sortable],
+      },
+      {
+        title: t('public~Kind'),
+        sortField: 'kind',
+        transforms: [sortable],
+      },
+      {
+        title: t('public~Namespace'),
+        sortField: 'metadata.namespace',
+        transforms: [sortable],
+      },
+    ];
+  };
+  const Row = (rowArgs: RowFunctionArgs<K8sResourceKind>) => {
+    const { obj, index, key, style, customData } = rowArgs;
+    const { csvName, cancel } = customData;
+    return (
+      <TableRow id={obj.metadata.uid} index={index} trKey={key} style={style}>
+        <TableData>
+          <OperandLink obj={obj} csvName={csvName} cancel={cancel} />
+        </TableData>
+        <TableData className="co-break-word" data-test-operand-kind={obj.kind}>
+          {obj.kind}
+        </TableData>
+        <TableData>
+          {obj.metadata.namespace && (
+            <ResourceLink
+              kind="Namespace"
+              title={obj.metadata.namespace}
+              name={obj.metadata.namespace}
+              onClick={cancel}
+            />
+          )}
+          {!obj.metadata.namespace && 'None'}
```

Comment:
Use `-` instead which is what we use in other contexts. Prefer a ternary here.

Reply:
fixed